src/util-classification-config.c

```diff
@@ -697,22 +708,15 @@ static int SCClassConfTest02(void)
 static int SCClassConfTest03(void)
 {
     DetectEngineCtx *de_ctx = DetectEngineCtxInit();
-    int result = 0;
 
-    if (de_ctx == NULL)
-        return result;
+    FAIL_IF_NULL(de_ctx);
 
     FILE *fd = SCClassConfGenerateInValidDummyClassConfigFD02();
-    SCClassConfLoadClassficationConfigFile(de_ctx, fd);
-
-    if (de_ctx->class_conf_ht == NULL)
-        return result;
-
-    result = (de_ctx->class_conf_ht->count == 3);
```

Comment:
Looks like this check was removed, why ?

Reply:
Bur are the functionalities still tested somehow ?

Reply:
Yes and no. Loading of the config file will fail since in unittest mode so there is no longer partial state maintained.

Reply:
What is the partial state ? `class_conf_ht` ? Should it be completely removed ?

Reply:
No; it's still used but no longer a measure of how many file lines failed parsing as failure occurs on the first parse failure.